examples/intermediate/mplot3d.py

```diff
@@ -24,7 +24,7 @@ def mplot3d(f, var1, var2, show=True):
     p = import_module('pylab')
     # Try newer version first
     p3 = import_module('mpl_toolkits.mplot3d',
-        __import__kwargs={'fromlist': ['something']}) or import_module('matplotlib.axes3d')
+        import_kwargs={'fromlist': ['something']}) or import_module('matplotlib.axes3d')
```

Comment:
In mypy an argument that begins with two underscores is a "positional-only" argument.